CODEOWNERS

```diff
@@ -14,6 +14,7 @@
 /src/Lean/Data/Lsp/ @mhuisi
 /src/Lean/Elab/Deriving/ @semorrison
 /src/Lean/Elab/Tactic/ @semorrison
+/src/Lean/Language/ @Kha
```

Comment:
Very minor style nitpick: Did you consider `Lean.Lang` for the module prefix? It feels like it would fit better with the other 4-letter prefixes (e.g., `Data`, `Elab`, `Meta`, `Util`) and match the eventual `#lang`.

Reply:
I didn't but it sounds like a good idea

Reply:
Let's stay with `Language`, `#lang` will likely be `language` anyway